### PR TITLE
adding hash fuzz targets

### PR DESCRIPTION
**Pull request by mattjurenka**
My submission for new integration of the MystenLabs/fastcrypto repository.

### CODE REVIEW

File: .github/workflows/mayhem.yml [hunk 1]
```diff
@@ -0,0 +1,112 @@
+
```
Comment (xansec) [inline .github/workflows/mayhem.yml]: Could you please use a matrix action to run these analyses instead? For example https://github.com/mayhemheroes/mayhem-cmake-example/blob/solution/.github/workflows/mayhem.yml#L61-L62

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: .github/workflows/mayhem.yml (hunk 1 above)
Comment (mattjurenka) [reply .github/workflows/mayhem.yml]: Just made the required changes. Will shortly update my other submissions to use this mayhem.yml format as well

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**